test/MOIWrapper.jl

```diff
@@ -24,6 +24,7 @@ end
         # linear1 test is disabled due to the following bug:
         # https://projects.coin-or.org/Clp/ticket/84
         "linear1",
+        "linear8c",
```

Comment:
This is the list of excluded tests. We want to run this test.

Reply:
oh got it